statler_site/templates/play-list-page.html

```diff
@@ -14,16 +14,23 @@
       $(document).ready(function(){
         // Get JSON from a URL as an object
         $.getJSON(apiEndpointUrl, function(result){
-          // display the returned json
-          $('p').append(JSON.stringify(result));
+          // display the returned json dynamically
+          $.each(result, function(key, value){
+            $("<li/>").appendTo("#plays").html("<h3>" + value.fields.title + "</h3");
+            $("<p/>").appendTo("#plays").html(value.fields.title);
+            $("<p/>").appendTo("#plays").html(value.fields.director);
```

Comment:
It's sub-optimal that we need to dot-into the "fields" object. We should fix this on the API side. (Relevant to #35, #38, #39)